core/renderers/common/constants.js

```diff
@@ -37,6 +37,13 @@ goog.require('Blockly.utils.userAgent');
  */
 Blockly.blockRendering.ConstantProvider = function() {
 
+  /**
+   * Whether to add a 'hat' on top of all blocks with no previous or output
+   * connections. Can be overridden by 'hat' property on Theme.BlockStyle.
+   * @type {boolean}
+   */
+  this.ADD_START_HATS = false;
```

Comment:
These constants aren't currently alphabetically sorted. Can we add this close to the START_HAT_HEIGHT and START_HAT_WIDTH.

Reply:
Done.